app/shared/experiments/FeatureFlagsProvider.js

```diff
@@ -124,7 +107,8 @@ export const FeatureFlagsProvider = ({ children }) => {
       refresh,
       clear: () => client?.clear(),
     }
-  }, [ready, refresh])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why are we disabling eslint for this line? If the deps are incomplete, we should prolly correct that, no?

Reply:
The comment line is still there lol

Reply:
oh I removed the other one. fixed.